### ISSUE TITLE
RangeError: Invalid time value

### ISSUE THREAD

**Issue by nodegin** [felixmosh/bull-board#839]
When I click into "DELAYED 1" tab of the queue,
I am encountering this error:

<img width="1494" alt="2024-10-16 11 43 54" src="https://github.com/user-attachments/assets/4bfb21d1-383c-4935-bad8-4365017da3d7">

<img width="972" alt="2024-10-16 11 52 11" src="https://github.com/user-attachments/assets/f1c734a2-4195-45d5-a926-85e2f0e24ec7">

**Comment by felixmosh** [comment]
Hi, thank you for reporting this bug, can you elaborate a bit,
Which queue are you using? Bull or BullMQ
Which server adapter?
What is you job data & options?

How can I reproduce this?

**Comment by Nekkiton** [comment]
HI, I have the same issue after migrting from 5.14.0 to 6.2.1 version
I'm using NestJs+bullmq
ExpressAdapter


This issue happens only for one queue from 20. The only difference for this queue is that its worker has concurrency property

Below you can see the data I've got for `/api/queues?activeQueue=Agent.disconnected&status=delayed&page=1&jobsPerPage=10` request

 ```
{
      "name": "Agent.disconnected",
      "statuses": [
        "latest",
        "active",
        "waiting",
        "waiting-children",
        "prioritized",
        "completed",
        "failed",
        "delayed",
        "paused"
      ],
      "counts": {
        "active": 0,
        "completed": 41,
        "delayed": 1,
        "failed": 1,
        "paused": 0,
        "prioritized": 0,
        "waiting": 0,
        "waiting-children": 0
      },
      "jobs": [
        {
          "id": "54",
          "timestamp": 1729072848292,
          "processedOn": 1729073751539,
          "progress": 0,
          "attempts": 16,
          "delay": 60000,
          "failedReason": "The agent is still unavailable",
          "stacktrace": [
            // some stacktrace
          ],
          "opts": {
            "attempts": 4320,
            "removeOnFail": 10000,
            "removeOnComplete": 10000,
            "backoff": {
              "delay": 60000,
              "type": "fixed"
            }
          },
          "data": {
            // some data
          },
          "name": "Agent.disconnected",
          "returnValue": null,
          "isFailed": true
        }
      ],
      "pagination": {
        "pageCount": 1,
        "range": {
          "start": 0,
          "end": 9
        }
      },
      "readOnlyMode": false,
      "allowRetries": true,
      "allowCompletedRetries": true,
      "isPaused": false,
      "type": "bullmq"
    }
```

**Comment by felixmosh** [comment]
I've injected your data, it works for me...
<img width="749" alt="image" src="https://github.com/user-attachments/assets/46c9f3bd-e9f2-4f85-b45c-715728175c80">

**Comment by Nekkiton** [comment]
I see that issue only on "DELAYED" tab, other tabs works great

**Comment by felixmosh** [comment]
@Nekkiton can you provide the job options that you are creating a job with?

**Comment by Nekkiton** [comment]
quite strange I use next job options:
```
           "attempts": 4320,
           "removeOnFail": 10000,
           "removeOnComplete": 10000,
           "backoff": {
             "delay": 60000,
             "type": "fixed"
           }
```
And that I see:
![screen](https://github.com/user-attachments/assets/9ee45a1a-5913-45ce-b2d5-3ef8da5dbad8)

**Comment by felixmosh** [comment]
This is weird, why It get added with the `delayed` status? there is no `delay` property in your options...
When I add a job with your options, it get started immediatetly...


https://github.com/user-attachments/assets/edb172de-fbb3-46a0-8826-057a7aa1f92b

**Comment by Nekkiton** [comment]
> why It get added with the delayed status? there is no delay property in your options...

this is because of custom logic inside the processor. It failed almost immediately since I didn't pass necessary data

> Can you double check that you are using bullMQ adapter with the BullMQ queue (or the Bull Adapter with the Bull Queue)

Yes, I've checked that right now. everything is ok, in general queues works as expected, only "delayed" tab on ui doesn't work :(

**Comment by felixmosh** [comment]
It is still works for me... :|
I've added a `throw` in the job processor, it gets delayes, as it is on your video, but everything just works.

Can you open the network tab, and provide me the JSON?
This is a client side error, which throws when some number is null (probably).

**Comment by Segfaultd** [comment]
Can confirm same issue here. Happy that you fixed it. Any idea about a release timeframe?

**Comment by felixmosh** [comment]
@Nekkiton thank you for that json, it helped me to track the issue... (not sure how it can happend)...
Fixed in `v6.2.4`

**Comment by Nekkiton** [comment]
Awesome! Thank you for quick fix here
